controllers/api.articles.controller.js

```diff
@@ -1,10 +1,44 @@
 const { fetchArticles } = require("../models/api.articles.model");
 
+const validSortByColumns = [
+  "article_id",
+  "title",
+  "author",
+  "topic",
+  "created_at",
+  "votes",
+];
+const validOrders = ["asc", "desc"];
+
 exports.getArticles = (req, res, next) => {
-  fetchArticles().then((articles) => {
-    res.status(200).send({articles});
-  }) .catch((err) => {
-    next(err);
-  });
+  if (req.query.sort_by) {
+    sort_by = req.query.sort_by;
+  } else {
+    sort_by = "created_at"
+  }
+  if (req.query.order) {
+    order = req.query.order
+  } else {
+    order = "desc"
+  }
+  
```

Comment:
You could default these values by defaulting the parameters that get passed to getArticles. If the value exists and is passed through it will use it - if not it will default to what you set it. Worth a google, it would mean you won't need these if/elses!